manifests/c/com/neosubhamoy/pytubepp/helper/0.7.0/com.neosubhamoy.pytubepp.helper.installer.yaml

```
@@ -0,0 +1,20 @@
+# Created with komac v2.10.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: com.neosubhamoy.pytubepp.helper

```

```suggestion
PackageIdentifier: neosubhamoy.pytubepp.helper.beta
```
Please update file name and path to match this change.